src/raqm.c

```diff
@@ -1507,6 +1548,10 @@ _raqm_resolve_scripts (raqm_t *rq)
 static bool
 _raqm_shape (raqm_t *rq)
 {
+  hb_buffer_flags_t hb_buffer_flags = HB_BUFFER_FLAG_BOT | HB_BUFFER_FLAG_EOT;
+  if (rq->invisible_glyph < 0)
+    hb_buffer_flags |= HB_BUFFER_FLAG_REMOVE_DEFAULT_IGNORABLES;
```

Comment:
`HB_BUFFER_FLAG_REMOVE_DEFAULT_IGNORABLES` is rather new as well. I think we can just `#ifndef … #define` it, and it would do nothing on older versions of HarfBuzz.

Reply:
Added an autoconf check for it.